SPECS/nfs-utils/nfs-utils.spec

```diff
@@ -125,10 +125,10 @@ make check
 
 %pre
 if ! getent group nobody >/dev/null; then
-    groupadd -r nobody
+    groupadd -r -g 65534 nobody
```

Comment:
Allied comment .. similar to `mysql` change that we did the other day:
- https://github.com/microsoft/CBL-Mariner/pull/7496

I think we should add a dependency on `shadow-utils` :)

Reply:
Hey @mfrw i can see the useradd dependency already called out in the spec file

Requires(pre):  /usr/sbin/useradd /usr/sbin/groupadd
Requires(postun):/usr/sbin/userdel /usr/sbin/groupdel

Reply:
Sorry my bad .. Missed it.